mudz.Common/Domain/Environment/Map/Room/RoomCollection.cs

```diff
@@ -0,0 +1,14 @@
+﻿using mudz.Common.Domain.Player;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace mudz.Common.Domain.Environment.Map.Room
+{
+	public sealed class RoomCollection : Dictionary<RoomKey, RoomContent>
```

Comment:
Just splitting out some logic and making things more readable (plus typing player instead of the string throughput)